common/src/main/java/xyz/wagyourtail/jsmacros/client/api/sharedclasses/PositionCommon.java

```diff
@@ -50,6 +50,27 @@ public Pos2D add(double x, double y) {
             return new Pos2D(this.x+x, this.y+y);
         }
 
+        /**
+         * @param pos the position to subtract
+         * @return the new position.
+         *
+         * @since 1.8.4
+         */
+        public Pos2D sub(Pos2D pos) {
+            return new Pos2D(x - pos.x, y - pos.y);
```

Comment:
sub is the same as add(pos.multiply(-1)), is it really nececairy?
(also maybe there should be a `.negate()`)

Reply:
I mean subtract is a really common operation and `add(pos.multiply(-1))` is a bit to long in my opinion for such a simple thing.
As for the negate method, it certainly doesn't hurt to add it, but that's a thing you can do if you want to.
I would suspect that it would have been much more useful without the `sub` method.

Reply:
> I mean subtract is a really common operation and `add(pos.multiply(-1))` is a bit to long in my opinion for such a simple thing.

The same goes for the divide methods which could be done by multiplying with the reciprocal of the number.

Reply:
yeah. I guess they're fine to keep